omnibus/resources/puppy/msi/bundle.wxs.erb

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"
+      xmlns:bal="http://schemas.microsoft.com/wix/BalExtension">
+  <?include "parameters.wxi" ?>
+	<Bundle Name="Datadog Agent Installer" 
```

Comment:
Should we make a difference between the full Agent and the puppy Agent? Right now the two bundles have the same name